hopp/simulation/technologies/hydrogen/h2_storage/lined_rock_cavern/lined_rock_cavern.py

```diff
@@ -0,0 +1,158 @@
+"""
+Author: Kaitlin Brunik
+Created: 7/20/2023
+Institution: National Renewable Energy Lab
+Description: This file outputs capital and operational costs of lined rock cavern hydrogen storage.
+It needs to be updated to with operational dynamics.
+Costs are in 2018 USD
+
+Sources:
+    - [1] Papadias 2021: https://www.sciencedirect.com/science/article/pii/S0360319921030834?via%3Dihub
+    - [2] Papadias 2021: Bulk Hydrogen as Function of Capacity.docx documentation at hopp/hydrogen/h2_storage
+    - [3] HDSAM V4.0 Gaseous H2 Geologic Storage sheet
+"""
+
+import numpy as np
+from hopp.simulation.technologies.hydrogen.h2_transport.h2_compression import Compressor
+
+class Lined_Rock_Cavern_Storage():
```

Comment:
I recommend changing this to `LinedRockCavernStorage` per Python convention https://peps.python.org/pep-0008/#class-names